docs/source/operations/howto/backend-deployment.rst

```diff
@@ -0,0 +1,134 @@
+*****************************
+How to deploy Substra Backend
+*****************************
+
+This guide shows you how to deploy the backend component of Substra.
+
+Prerequisites
+=============
+
+To deploy a Substra backend you will need a fully configured Kubernetes cluster.
+You will also need to install `Helm <https://helm.sh/>`_.
+
+Preparing your Helm values
+==========================
+
+The Substra backend deployment is packaged using Helm.
+You can find the complete description of values that can be used to configure the chart on `Artifact Hub <https://artifacthub.io/packages/helm/substra/substra-backend>`_.
+
+To configure your values:
+
+#. Add the Helm repository for Substra:
+
+   .. code-block:: bash
+
+      helm repo add substra https://substra.github.io/charts/
+
+#. Create a Helm values file named ``backend-values.yaml`` with the following content:
+
+   .. code-block:: yaml
+
+      organizationName: ORGANIZATION
+
+   | Replace ``ORGANIZATION`` with the name of your :term:`Organization`.
+     It should match one of the organizations present in the :ref:`Orchestrator channel configuration <orchestrator-channel-config>`.
+
+#. Configure your Substra backend Ingress. In the ``backend-values.yaml`` file add the following content:
+
+   .. code-block:: yaml
+
+      config:
+        ALLOWED_HOSTS: '[".HOSTDOMAIN"]'
+
+      server:
+        defaultDomain: https://SUBDOMAIN.HOSTDOMAIN:443
+        commonHostDomain: HOSTDOMAIN
+
+        ingress:
+          enabled: true
+          hostname: SUBDOMAIN.HOSTDOMAIN
+   
+   | Replace ``HOSTDOMAIN`` with the domain of your server without the lowest subdomain (e.g. for a server exposed at ``api.substra.org`` ``HOSTDOMAIN`` would be ``substra.org``).
+   | Replace ``SUBDOMAIN`` with the lowest subdomain (e.g. for a server exposed at ``api.substra.org`` ``SUBDOMAIN`` would be ``api``).
+
+#. Configure your connection to the orchestrator. In the ``backend-values.yaml`` file add the following content:
+
+   .. code-block:: yaml
+
+      orchestrator:
+        host: ORCHESTRATOR_HOSTNAME
+        port: ORCHESTRATOR_PORT
+        mspID: ORGANIZATION
+
+   | Replace ``ORCHESTRATOR_HOSTNAME`` with the hostname of the orchestrator.
+   | Replace ``ORCHESTRATOR_PORT`` with the port of your orchestrator (Should be ``80`` if TLS is disabled, otherwise ``443``).
+   | Replace ``ORGANIZATION`` with the name of your Organization. It should be the same value as for the ``organizationName`` key.
```

Comment:
Note for later: we should eventually remove this `orchestrator.mspID` value if it's the same than the top level `organization`.

Reply:
It can be different but for simplicity it's almost always the same.